<launch>
    <arg name="port" />
    <arg name="baud" default="57600" />
    <arg name="mode_right" default="true" />
    <arg name="direction" default="1" />

    <node
        pkg="rosserial_python"
        type="serial_node.py"
        name="eye_display"
        output="screen"
        >
        <rosparam subst_value="true">
            port: $(arg port)
            baud: $(arg baud)
            mode_right: $(arg mode_right)
            direction: $(arg direction)
        </rosparam>
    </node>
</launch>
